WorkloadManagementSystem/Agent/PilotMonitorAgent.py

```diff
@@ -33,7 +33,7 @@ def initialize( self ):
     self.clearPilotsDelay = self.am_getOption( 'ClearPilotsDelay', 30 )
     self.clearAbortedDelay = self.am_getOption( 'ClearAbortedPilotsDelay', 7 )
 
-    self.WMSAdministrator = WMSAdministratorHandler()
+    self.WMSAdministrator = RPCClient('WorkloadManagement/WMSAdministrator')
```

Comment:
Could you please check with pylint if there are errors and add 
e.g., 
`#pylint disable=no-member`
for the clearPilots call

Reply:
My pylint isn't signalling anything in clearPilots call (similar situation in client using rpc to call service).

Reply:
Ok, thanks for checking.